example/gcs/src/main.rs

```diff
@@ -1,9 +1,9 @@
-mod storage_v1_types;
-
 use anyhow::Context;
-use async_google_apis_common as common;
+use async_google_apis as common;
 use env_logger;
 
+use common::storage_v1_types;
```

Comment:
same question as previously

Reply:
Because in new project structure, generated codes are sub module of a-g-a.